### PR TITLE
Update msal-angular tooling

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR updates the msal-angular tooling to use angular v11, the latest stable version of Angular.

### CODE REVIEW

File: lib/msal-angular/package.json [hunk 1]
```diff
@@ -21,40 +21,37 @@
...
     "rxjs": "~6.5.5",
     "ts-node": "~8.3.0",
     "tslib": "^2.0.0",
-    "typescript": "~3.9.5",
+    "typescript": "~4.1.5",
     "zone.js": "~0.10.3"
   },
   "peerDependencies": {
-    "@angular/common": "9 - 12",
-    "@angular/core": "9 - 12",
```
Comment (jasonnutter) [inline lib/msal-angular/package.json:57]: Why was this removed? (this essentially documents which versions of Angular we support)

**Review by jasonnutter — APPROVED** [review]

**Review by tnorling — APPROVED** [review]
